File: algorithm/src/main/java/baekjoon/p01076/Main.java
package baekjoon.p01076;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String[] inputs = new String[3];

        for (int i = 0; i < inputs.length; i++) {
            inputs[i] = sc.next();
        }

        System.out.println(solve(inputs));
    }

    private static long solve(String[] inputs) {
        String[] set = {
                "black",
                "brown",
                "red",
                "orange",
                "yellow",
                "green",
                "blue",
                "violet",
                "grey",
                "white",
        };

        int a = idx(set, inputs[0]);
        int b = idx(set, inputs[1]);
        int c = idx(set, inputs[2]);

        return (a * 10 + b) * (long) (Math.pow(10, c));
    }

    private static int idx(String[] arr, String str) {
        for (int i = 0; i < arr.length; i++) {
            if (arr[i].equals(str)) {
                return i;
            }
        }

        return -1;
    }

}

File: algorithm/src/main/java/baekjoon/p01076/Main.kt
package baekjoon.p01076

import kotlin.math.pow

fun main(args: Array<String>) {
    val inputs = Array(3) { readLine()!! }
    println(solve(inputs))
}

private fun solve(inputs: Array<String>): Long {
    val set = arrayOf(
            "black",
            "brown",
            "red",
            "orange",
            "yellow",
            "green",
            "blue",
            "violet",
            "grey",
            "white"
    )

    val a = set.indexOf(inputs[0])
    val b = set.indexOf(inputs[1])
    val c = set.indexOf(inputs[2])

    return (a * 10 + b) * (10f.pow(c)).toLong()
}